bin/metrics-redis-graphite.rb

```diff
@@ -97,6 +97,12 @@ class Redis2Graphite < Sensu::Plugin::Metric::CLI::Graphite
          long: '--skipkeys KEYS',
          default: nil
 
+  option :conn_failure_status,
+         long: '--conn-failure-status EXIT_STATUS',
+         description: 'Exit status for Redis connection failures',
+         default: 'unknown',
+         in: %w(unknown warning critical ok)
```

Comment:
I question the value of `ok` as an option...A connection failure is clearly not OK...